Tidy preface chapter wording (#3)
biscoffhamster/third-cage-on-the-left: book/01-preface.qmd

North London', but instead I said that from the age of one to six we lived in Regent's Park before moving
to NW2.

> 'Whereabouts in Regent's Park?' she asked. I then explained how, when I was just a year old, my father
> returned from Africa to take up a post with the grand-sounding title of Superintendent of London Zoo, and

Author: This change does three things in one place: it adds the missing question mark, improves the sentence rhythm with commas and a hyphen, and fixes the spelling of `Superintendent`. The intent was to keep the meaning unchanged while making the line read more cleanly as book prose.

If you want to preserve the imported wording exactly, you can revert it with this suggestion.

```suggestion
'Whereabouts in Regent's Park' she asked. I then explained how when I was just a year old my father
returned from Africa to take up a post with grand sounding title of Superintendant of London Zoo, and
```

that our house was actually in the middle of the zoo itself.

She was incredulous but started asking me how much I could remember about my life in the zoo. 'Well
to be honest, I can remember a lot, as if it was just yesterday in fact.' It was then as we talked she started
saying what a wonderful story it could make. Curiously it was a few years before the rather bizarre story

> Life of Pi was published, but I did admit it was the most wonderful environment for a small child to

Author: This change removes the leading `The` from `The life of Pi`. I treated that as an import artifact rather than an intentional stylistic choice, since the title reads more naturally here as `Life of Pi` in running prose.

If you want to keep the imported phrasing, you can revert it with this suggestion.

```suggestion
The life of Pi was published, but I did admit it was the most wonderful environment for a small child to
```

grow up in. I can trace my interest in animals and the natural environment to those early years and didn't
even mind the taunts of my school friends about living in the monkey house. I put them down to
jealousy.

David, my brother is two years older than me, so after he started school I was on my own at home. I'm
pretty sure if my mother was to have done today what she did then she would have been in big trouble
with Social Services, but she was happy for me to roam the zoo by myself on my tricycle during the

> mornings, with a note pinned to my back saying "PLEASE SEND ME HOME AT 12 O'CLOCK FOR LUNCH".

Author: This change normalizes the quoted lunch note by fixing the apostrophe/capitalization and by adding the closing full stop for the surrounding sentence. The wording of the note itself is otherwise unchanged.

If you want to retain the imported casing and punctuation, you can revert it with this suggestion.

```suggestion
mornings, with a note pinned to my back saying "PLEASE SEND ME HOME AT 12'o Clock for LUNCH"
```

I came to no harm, or at least not very often. My trike had no brakes, and if you have ever been to
London Zoo you may remember how it is divided in two by a road (The Outer Circle). Two tunnels allow

> the public to access both sides of the zoo, so for me it was great sport to come round the corner, then

Author: This is a straightforward grammar fix: `both side` became `both sides`. It does not alter tone or meaning, only corrects the imported text.

If you want to keep the imported wording unchanged, you can revert it with this suggestion.

```suggestion
the public to access both side of the zoo, so for me it was great sport to come round the corner, then
```

with legs off the pedals and wide apart, whizz down the tunnel at top speed. That was all very well until
one day I came round the corner to see the tunnel was full of people. I aimed for a small gap to the left -
see, I remember it well - but hit the concrete wall instead. Luckily the zoo's electric meat wagon came
upon me, scraped me up and to the first aid post.

> Apparently apart from a bent nose, which I have to this day I was relatively unscathed.

Author: This deletion removes the social-post style sign-off from the end of the preface. My reasoning was that it reads like the original Facebook or email framing rather than part of the memoir itself, so the chapter ends more cleanly without it.

If you want to keep that original closing in the book text, you can revert it with this suggestion.

```suggestion
Apparently apart from a bent nose, which I have to this day I was relatively unscathed.

* * *

Well, dear friends, that is my two penny worth for starters. If the bug gets me that might be all you'll ever
get, and I will forgive you for thinking 'thank goodness for that' but much as I love Bonny her
conversation is somewhat limited, so this project might keep me relatively sane.
```
